libs/ui/src/components/activity-section-list/activity-section-list.tsx

```diff
@@ -112,15 +154,17 @@ export const ActivitySectionList: FC<Props> = ({ publicKeyHash, chainId, filterT
 
   return (
     <SectionList
+      ref={sectionListRef}
       sections={activity}
       renderItem={renderItem}
       renderSectionHeader={renderSectionHeader}
+      initialNumToRender={isMobile || offsetY === 0 ? 5 : 3}
```

Comment:
as I understand, `initialNumToRender` using only in initial render.
is it mean `offsetY === 0` is always true in this case?

Reply:
it's mean that we need to render 5 items if we are on a Mobile device or we have offset === 0 on the Web, in other cases, we need to render just 3 items

Reply:
yes, I understand
but my question is: Will `offset === 0` be always true in `initialNumToRender` ?

Reply:
I think the `offset` will have different values for `initialNumToRender`, or I miss understand your question)

Reply:
<img width="835" alt="Screenshot 2023-02-09 at 12 55 23" src="https://user-images.githubusercontent.com/58780063/217793601-79419c81-e3b1-4db9-a7e3-00b09f05c8dd.png">

we change initial `offset` only with `onScroll` func..

Reply:
updated on refactoring